modules/page_base.py

```diff
@@ -616,8 +642,7 @@ def hide_popup_by_child_node(
                  if (element && element.hidePopup) {
                     element.hidePopup();
                  }"""
-        with self.driver.context(self.context_id):
-            self.driver.execute_script(script, node)
+        self.driver.execute_script(script, node)
```

Comment:
I just wanted to know why the driver management (`with self.driver.context(self.context_id)` line) was removed

Reply:
It actually is what made my test pass--looks like that for all cases where we thought we had to send the JS command to the chrome context, the content context would have been fine, but for this instance, the chrome context was not responsive